### PR TITLE
Removed not needed return statements

### PR DESCRIPTION
**Pull request by marcusnaslund**
For consistency.

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Anything to say about this @davidhesselbom ?

File: source/draw/RasterImage.ooc [hunk 1]
```diff
@@ -23,7 +23,7 @@ import Color
 import StbImage
 
 RasterImage: abstract class extends Image {
-	distanceRadius: Int { get { return 1; } }
+	distanceRadius: Int { get { 1 } }
```
Comment (davidhesselbom) [inline source/draw/RasterImage.ooc]: While you're at it, can't this be `distanceRadius ::= 1`?

Comment (marcusnaslund) [inline source/draw/RasterImage.ooc]: `::=` is read-only, right?

Comment (davidhesselbom) [inline source/draw/RasterImage.ooc]: So is `get` with no `set`.

Comment (marcusnaslund) [inline source/draw/RasterImage.ooc]: Thanks. I will change that in some subclasses as well.

**Comment by davidhesselbom** [comment]
Looks good to me, but I don't have the time to validate it.

**Comment by marcusnaslund** [comment]
> Looks good to me, but I don't have the time to validate it.

Oh you don't have to, just wanted another pair of eyes. Thanks.

**Pull request merged**